libcontainer/configs/mount_linux.go

```diff
@@ -2,6 +2,24 @@ package configs
 
 import "golang.org/x/sys/unix"
 
+type MountMapping struct {
+	// Recursive indicates if the mapping needs to be recursive.
+	Recursive bool `json:"recursive"`
+
+	// NamespacePath is a path to a user namespace that indicates the necessary
+	// id-mappings for MOUNT_ATTR_IDMAP. If set to non-"", UIDMappings and
+	// GIDMappings must be set to nil.
+	NamespacePath string `json:"userns_path,omitempty"`
```

Comment:
```suggestion
	NamespacePath string `json:"namespace_path,omitempty"`
```
(Or rename the Go variable to match the json field name)